pkg/operator/ceph/csi/operator_driver.go

```diff
@@ -175,7 +175,7 @@ func (r *ReconcileCSI) createOrUpdateCephFSDriverResource(cluster cephv1.CephClu
 
 func (r *ReconcileCSI) createOrUpdateNFSDriverResource(cluster cephv1.CephCluster, clusterInfo *cephclient.ClusterInfo) error {
 	resourceName := fmt.Sprintf("%s.nfs.csi.ceph.com", r.opConfig.OperatorNamespace)
-	spec, err := r.generateDriverSpec(cluster)
+	spec, err := r.generateDriverSpec(cluster, "nfs")
```

Comment:
why not added it below after generating? instead of passing known string and adding it?